app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2008-2010 Jonathan Beck
+	2009-2014 Nikias Bassen
+	2009-2014 Martin Szulecki
+	2008 Matt Colyer"
+LICENSE="GNU GPL v2
+	GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.libimobiledevice.org/downloads/libplist-$portVersion.tar.bz2"
+CHECKSUM_SHA256="0effdedcb3de128c4930d8c03a3854c74c426c16728b8ab5f0a5b6bdc0b644be"
+
+ARCHITECTURES="x86 !x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	libplist$secondaryArchSuffix = $portVersion compat >= 1
+	cmd:plistutil = $portVersion compat >= 1
+	lib:libplist$secondaryArchSuffix = 3.0.0 compat >= 1
+	lib:libplist++$secondaryArchSuffix = 3.0.0 compat >= 1
```

Comment:
Oops, you also changed **`compat >= 3`** into **`compat >= 1`**. Only the **`compat >= 0`** were to be changed to **`compat >= 1`**.
Well, since this is a bit confusing, I've uploaded the recipe to my remote branch:
https://github.com/fbrosson/haikuports/blob/libplist/app-pda/libplist/libplist-1.12.recipe
BTW, you also changed the indentation in BUILD and it is not very common. Could you copy the indentation I put in my branch? Thanks!